image-encapsulate.cpp

```diff
@@ -2,13 +2,12 @@
 #include "brightener.h"
 
 int main() {
-	auto image = std::make_unique<Image>(512, 512);
+	auto image = std::make_shared<Image>(512, 512);
 	std::cout << "Brightening a 512 x 512 image\n";
 	ImageBrightener brightener(std::move(image));
```

Comment:
`move` is not needed with `shared_ptr`. The idea of a shared pointer is to share ownership of the object, so we don't have to "move" its responsibility from one object to another.

Reply:
Taken care